doc/man7/openssl-threads.pod

```diff
@@ -0,0 +1,100 @@
+=pod
+
+=head1 NAME
+
+openssl-threads - Overview of thread safety in OpenSSL
+
+=head1 DESCRIPTION
+
+In this man page, we use the term B<thread-safe> to indicate that an
+object can be used by multiple threads at the same time.
```

Comment:
I'd rather say "indicate that an object or function can be used". Thread safety is about more than just objects.

Reply:
added "object *or function* can be used"